src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3222,16 +3222,17 @@ namespace EvaporativeCoolers {
             if (EvapCond(EvapCoolIndex).FanPowerModifierCurveIndex > 0) {
                 FanPowerModCurveValue = CurveValue(EvapCond(EvapCoolIndex).FanPowerModifierCurveIndex, FlowRatio);
             } else {
-                FanPowerModCurveValue = EvapCond(EvapCoolIndex).PartLoadFract;
+                // linearly scale fan power using part-load-fraction and fan flow ratio when fan power modifier curve is not specified
+                FanPowerModCurveValue = EvapCond(EvapCoolIndex).PartLoadFract * FlowRatio;
```

Comment:
This should now be multiplied by FanPLR, not FlowRatio, correct?

Reply:
Actually this indirect research special calculation confused me a lot, this `FlowRatio` is the secondary fan flow ratio, so I think pump might need to be consistent with secondary fan?

Reply:
If this is the indirect model then maybe the power calculation should not change until it is better understood.